docs/source/user-guide/compatibility.md

```diff
@@ -93,104 +97,104 @@ Cast operations in Comet fall into three levels of support:
 
 - **Compatible**: The results match Apache Spark
 - **Incompatible**: The results may match Apache Spark for some inputs, but there are known issues where some inputs
-will result in incorrect results or exceptions. The query stage will fall back to Spark by default. Setting
```

Comment:
All of the changes from this point are the result of running `prettier` to format the document